modules/data_release/README.md

```diff
@@ -1,16 +1,40 @@
-# Data Release Module
+# Data Release
 
-The Data Release Module can be used to easily distribute packaged data releases of your study.
+## Purpose
 
-## Features:
+The Data Release module can be used to easily distribute packaged data releases
+of the study.
 
-- Upload new releases through the front-end web interface
-- Differentiate releases using (optionally unique) versioning tag
-- Assign permissions for each release to any specific user
+## Intended users
 
-## Other notes:
+- Researchers and analysis teams
+- Data coordinator that will manage access to, and upload of, the data releases
 
-- Uploads are stored under the modules/data_release/user_uploads directory which can easily be symlinked to another location if necessary, but please create and make sure it is writable by your web server
-- Only superusers have permission to upload files and grant permissions
-- Remove permissions by deleting rows in the data_release_permissions table
-- Upload date will automatically be added during file upload
+## Scope
+
+The module is used to manage, upload and download data releases of the study.
+- Different data releases can be differentiated using the version tag.
+- Data coordinators can upload data releases of the study via the front-end
+web interface.
+- For each data release, data coordinators can manage permissions of any
+  specific user.
+- Upload date will automatically be added during file upload.
+
+
+## Permissions
+
+- Users permissions to view data releases are managed within the data release
+  module.
+- Only superusers have permission to upload and view all data releases.
```

Comment:
This is ambiguous the way that it's worded. Is this two different things? "(Only superusers have permission to upload) and (Only superusers have permission to view all)" or "Only superusers have permission to (upload and view) *all* data releases" and there are separate permissions for non-all variants? If the latter, it's unneeded since that's the definition of the superuser permission, if it's the former, then maybe there should be a ticket to separate uploading from the superuser permission..

Reply:
@Jkat can comment on this one. I believe you need to be a superuser to (upload and view) all data releases. If that is the case, I agree, we should create a specific permission for the data release module instead of making multiple users superusers.

Reply:
Only superusers can upload anything, no one else has upload permission
Only superusers can grant permissions - whether by "version" or specific "file_name", no one else has granting permissions
Once you are granted permission on any data release, it will show up in your list and you can click on it to download

Hope that's clear. Something like a "data_release_edit" permission can be added to avoid using the superuser permission.